cirq-core/cirq/transformers/analytical_decompositions/two_qubit_to_cz_test.py

```diff
@@ -257,3 +257,19 @@ def test_decompose_to_diagonal_and_circuit(v):
     combined_circuit = cirq.Circuit(cirq.MatrixGate(diagonal)(b, c), ops)
     circuit_unitary = combined_circuit.unitary(qubits_that_should_be_present=[b, c])
     cirq.testing.assert_allclose_up_to_global_phase(circuit_unitary, v, atol=2e-6)
+
+
+def test_remove_partial_czs_or_fail():
+    assert (
+        cirq.transformers.analytical_decompositions.two_qubit_to_cz._remove_partial_czs_or_fail(
+            [cirq.CZ(*cirq.LineQubit.range(2)) ** 1e-15], atol=1e-9
+        )
+        == []
+    )
+    with pytest.raises(ValueError):
+        _ = (
+            cirq.transformers.analytical_decompositions.two_qubit_to_cz._remove_partial_czs_or_fail(
+                [cirq.CZ(*cirq.LineQubit.range(2)) ** -0.5], atol=1e-9
+            )
```

Comment:
Since it will raise an error, you can probably get rid of the extra comparison and other stuff.

Reply:
done